tools/remark-preset-lint-node/index.js

```diff
@@ -0,0 +1,44 @@
+// @see https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md
+
+'use strict';
+
+module.exports.plugins = [
+  require('remark-lint'),
+  require('remark-lint-checkbox-content-indent'),
+  require('remark-lint-definition-spacing'),
+  require('remark-lint-fenced-code-flag'),
+  require('remark-lint-final-definition'),
+  require('remark-lint-final-newline'),
+  require('remark-lint-hard-break-spaces'),
+  require('remark-lint-no-auto-link-without-protocol'),
+  require('remark-lint-no-blockquote-without-caret'),
```

Comment:
It seems this is renamed: https://github.com/wooorm/remark-lint/tree/master/packages/remark-lint-no-blockquote-without-marker